packages/core/test/unit/sql/where.test.ts

```diff
@@ -1251,6 +1260,38 @@ Caused by: "undefined" cannot be escaped`),
         testSql({ intArrayAttr: { [Op.contains]: 1 } }, {
           default: new Error('1 is not a valid array'),
         });
+
+        testSql({
+          jsonAttr: { [Op.contains]: { foo: 'bar' } },
```

Comment:
You added this test to the range test suite. We have a lone `Op.contains` test in the JSONB suite. I think this should be moved there